src/blog/interact-with-your-audience-with-a-live-poll.md

```diff
@@ -0,0 +1,70 @@
+image: "https://cdn-images-1.medium.com/max/1600/1*VST3XVmoX1R3hAXJ8OGw4A.jpeg"
+---
+
+![](https://cdn-images-1.medium.com/max/1600/1*ULPgfU4_6DDEHDLDCdUMKQ.jpeg)
+
+*Photo by [Nicholas Green](https://unsplash.com/@nickxshotz?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText) on [Unsplash](https://unsplash.com/?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText)*
+
+We had this idea for a long time and it took us a bit of time to realize it, that’s why we are so excited to introduce today the new feature of [DeckDeckGo](https://deckdeckgo.com/), our web open-source editor for presentations, to make possible the interaction with your audience through live polling without having to leave your slides 🔥
+
+<iframe width="280" height="158" src="https://www.youtube.com/embed/GuF58XBzTj0" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe
+<br/>
+
+### User Experience
+
+UX could often be discussed and I’ve to say, in case of that particular feature, even more, as many solutions are feasible. I scratched my head for days, filling papers and taking notes until I finally figured out the core principles of the solution which were meaningful to me and hopefully to others too:
+
+* An audience could be a crowd, a small or large crowd. In case of live polling, not all participants are on the same page. Some might be fast to vote, some might be slow. Some people are near the projected slides or screen, others are probably faraway. Some might even be sleepy, some might have their phones in their bags, etc.
+
+* As a speaker I notably find convenient when I don’t have to switch between multiple applications, it could even to some extension makes me a bit less stressed. As an attendee, I tend to think that it needs me less effort to follow a presentation without to many media cut-offs. Summarized, I like when the presentation’s flow remains as smooth as possible.
+
+That’s why, at least in this first version of the features, all information are available at the same time and any time. Question, answers, connection information and even results of the vote, which are updated in real time, are all displayed on the same slide. Moreover, as I do think that QR codes are the best way to share links with the participants, the template also contains such a deep link to point them directly to the answers on their phones.
+
+![](https://cdn-images-1.medium.com/max/1600/1*6DlXoX4G2WZer9xVYymqug.gif)
+
+### Developer Mindset
+
+Since day one we always kept a developer mindset. That’s why both the editor and our developer starter kit are using the exact same core, the same web components as engine. Needless to say that it didn’t changed. This new feature isn’t just available in our editor but also available with our starter kit. If you rather like to develop your presentation using HTML or Markdown, it’s all cool and you have access to the exact same feature.
```

Comment:
```suggestion
Since day one we always kept a developer mindset. That’s why both the editor and our developer starter kit are using the exact same core, the same web components as engine. Needless to say that it didn’t change. This new feature isn’t just available in our editor but also available with our starter kit. If you would rather like to develop your presentation using HTML or Markdown, it’s all cool and you have access to the exact same feature.
```